examples/main.cpp

```diff
@@ -0,0 +1,29 @@
+#include "ThingooConnector.h"
+
+#include <ESP8266HTTPClient.h>
+#include <ArduinoJson.h>
+#include <WiFiManager.h>
+#include <WiFiClientSecureBearSSL.h>
+
+const char* fingerprint = "XX XX XX XX XX XX XX XX XX XX XX XX XX XX XX XX XX XX XX XX XX";
+const char* device_id = "device_1";
+const char* host = "website_address_without_https_or_http"; //example: www.google.com
```

Comment:
Actually it cannot start with www as it's related only to HTTP request and we will be using mqtt also. Just delete "www" prefix from example in the comment